src/main/java/com/google/sps/data/GivrUser.java

```diff
@@ -67,9 +73,16 @@ public boolean isLoggedIn() {
     return this.isLoggedIn;
   }
 
-  // TODO(): get correct moderator status for organization
-  public boolean isModerator(long organizationId) {
-      return false;
+  public ArrayList<Entity> getModeratingOrgs() {
+    setModeratingOrgs(); // Makes sure that getModeratingOrgs is returning updated information.
+    return this.moderatingOrgs;
+  }
+
+  public boolean isModeratorOfAnyOrg() {
+    if (getModeratingOrgs().size() > 0) {
```

Comment:
consider the following:

if (statement evaluates to true) {
  return true;
}
return false

vs

return statement